src/engine/enginedeck.cpp

```diff
@@ -43,6 +44,11 @@ EngineDeck::EngineDeck(const ChannelHandleAndGroup& handle_group,
         pEffectsManager->registerChannel(handle_group);
     }
 
+    // Make input_configured read-only.
+    m_pInputConfigured->connectValueChangeRequest(
+        this, SLOT(slotInputConfiguredChangeRequest(double)),
+        Qt::DirectConnection);
```

Comment:
My understanding of the docs says that if you find yourself forcing a slot connection type that you might be "doing it wrong." http://doc.qt.io/qt-4.8/threads-qobject.html says that with DirectConnection, "The slot is executed in the emitter's thread, which is not necessarily the receiver's thread." Is this what you want here (and elsewhere where you do this in this PR?)